src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+import { msg } from './intl'
+import './error_styles-nomodules.css'
```

Comment:
When importing a css, there is no need to use the `nomodules` tag.  The form to use when importing a CSS in the tag code is like this:
````jsx
import style from './error.css'

<div className={style['globalErrorContainer']}> ... </div>

<div className={`other-classes ${style['my-error-css-style']}`}> ... </div>
````